--- ChangeLog	2024-08-09 09:09:48.813412728 -0400
+++ ChangeLog-spell-corrected	2024-08-09 09:10:01.463349041 -0400
@@ -52,7 +52,7 @@
 2021-09-04  rocky <rb@dustyfeet.com>
 
 	* mathics_pygments/builtins.py,
-	mathics_pygments/data/mma-tables.json: Fix script capital N. Rebuld
+	mathics_pygments/data/mma-tables.json: Fix script capital N. Rebuild
 	tables
 
 2021-06-27  rocky <rb@dustyfeet.com>
@@ -73,7 +73,7 @@
 
 2021-06-22  R. Bernstein <rocky@users.noreply.github.com>
 
-	* : Merge pull request #2 from Mathics3/administrivia Varous lintings
+	* : Merge pull request #2 from Mathics3/administrivia Various lintings
 
 2021-06-22  rocky <rb@dustyfeet.com>
 
@@ -91,7 +91,7 @@
 2021-06-22  rocky <rb@dustyfeet.com>
 
 	* .github/workflows/autoblack.yml, .github/workflows/{osx.yaml =>
-	osx.yml}, .github/workflows/{ubuntu.yaml => ubuntu.yml}: Varous
+	osx.yml}, .github/workflows/{ubuntu.yaml => ubuntu.yml}: Various
 	lintings
 
 2021-06-22  rocky <rb@dustyfeet.com>
@@ -144,7 +144,7 @@
 
 2021-05-14  rocky <rb@dustyfeet.com>
 
-	* CHANGES.rst, MANIFEST.in, Makefile, mathics_pygments/version.py: 
+	* CHANGES.rst, MANIFEST.in, Makefile, mathics_pygments/version.py:
 	Get ready for release 1.0.0
 
 2021-05-14  rocky <rb@dustyfeet.com>
@@ -154,7 +154,7 @@
 
 2021-05-13  rocky <rb@dustyfeet.com>
 
-	* .gitignore, MANIFEST.in, mathics_pygments/builtins.py: 
+	* .gitignore, MANIFEST.in, mathics_pygments/builtins.py:
 	Administriva
 
 2021-05-12  rocky <rb@dustyfeet.com>
@@ -167,7 +167,7 @@
 
 	* mathics_pygments/builtins.py,
 	mathics_pygments/data/mma-tables.json, mathics_pygments/lexer.py,
-	mathics_pygments/style.py: More oeprators, what's up with Degree
+	mathics_pygments/style.py: More operators, what's up with Degree
 
 2021-05-09  rocky <rb@dustyfeet.com>
 
@@ -337,4 +337,3 @@
 2016-02-14  Ravi Menon <rsmenon@icloud.com>
 
 	* Initial commit for v0.1.0 of the Mathematica Lexer for Pygments
-
